src/main/scala/microsites/util/MicrositeHelper.scala

```diff
@@ -26,11 +26,43 @@ import microsites.util.FileHelper._
 import sbt._
 
 import scala.io.Source
+import com.sksamuel.scrimage._
 
 class MicrositeHelper(config: MicrositeSettings) {
   implicitly(config)
 
   val jekyllDir = "jekyll"
+  val faviconSizes = Seq(16,
+                         24,
+                         32,
+                         48,
+                         57,
+                         60,
+                         64,
+                         70,
+                         72,
+                         76,
+                         96,
+                         114,
+                         120,
+                         128,
+                         144,
+                         150,
+                         152,
+                         196,
+                         310).map { w =>
+    (w, w)
+  } ++ Seq((310, 150))
```

Comment:
I'd suggest doing this in this way:

```scala
@ val list = List(16,24,32,48,57,60,64,70,72,76,96,114,120,128,144,150,152,196,310,150)
list: List[Int] = List(16, 24, 32, 48, 57, 60, 64, 70, 72, 76, 96, 114, 120, 128, 144, 150, 152, 196, 310, 150)
@ list zip list
res1: List[(Int, Int)] = List((16, 16), (24, 24), (32, 32), (48, 48), (57, 57), (60, 60), (64, 64), (70, 70), (72, 72), (76, 76), (96, 96), (114, 114), (120, 120), (128, 128), (144, 144), (150, 150), (152, 152), (196, 196), (310, 310), (150, 150))
```

Also, do you really want a favicon size of `(310, 150)`?